src/components/CheatMode.tsx

```diff
@@ -21,18 +21,18 @@ interface CheatModeProps {
 }
 
 const CheatMode = ({ cheat }: CheatModeProps): ReactElement => {
-  const { data: cheatsEnabled } = useLoadAsync(getDevMode, null)
+  const { data: cheatsEnabled } = useLoadAsync(AsyncStorage.getDevMode, null)
   return (
     <>
       {cheatsEnabled && (
         <CheatContainer>
           <Button
-            label={labels.exercises.cheat.succeed}
+            label={getLabels().exercises.cheat.succeed}
```

Comment:
@SteffiStuffel why do we need a function for this btw? Just tried this and it seems to work as well (and would be way easier syntax to use it).
``` ts
export const labels: typeof importedLabels = importedLabels
```

Reply:
I imagine this leaves the opportunity to do some post-processing on the labels later on, should the need arise? Should I merge this for now, or revert back to `labels.`?

Reply:
Steffens solution is also fine for me. @charludo you can choose which one you prefer more.

Reply:
@SteffiStuffel @steffenkleinle in that case I'll leave it as is. Swapping `getLabels().` for `labels.` in the future is easy enough.